docs/misc/iaq.md

```diff
@@ -40,22 +87,12 @@ See [this post](https://www.mail-archive.com/qubes-devel@googlegroups.com/msg028
 
 MaxFPS, UseDirtyBits.
 
+## Tweaks
+
 ### Where are VM log files kept?
 
 In the `/var/log/libvirst/libxl/`, `/var/log/qubes/` and `/var/log/xen/console/` directories.
 
```

Comment:
I'd put this into the "troubleshooting" too

Reply:
hmm ; I clicked on the '+' symbol on the left of "where are VM log..." but for some reason the "tweaks" stuff is green. Maybe that's how it ought to be.